public/js/rules.js

```diff
@@ -0,0 +1,79 @@
+var Rule = React.createClass({
+    propTypes: {
+        name: React.PropTypes.string.isRequired,
+        contents: React.PropTypes.object.isRequired
+    },
+    render: function() {
+        var contentNodes = this.props.contents.map(function(c) {
+            return (
+                // Get raw data so any html formatting in the model isn't lost.
+                // eslint-disable-next-line react/no-danger
+                <li dangerouslySetInnerHTML={{__html:c.toString()}} />
```

Comment:
So that we can have rule contents with html inside of it. 
E.g. If you look at the "winning conditions" object in data/rules.json, the html in contents will be escaped automatically by react to be shown as "`<b>`two`</b>`" instead of "<b>two</b>" if we don't dangerouslySetInnerHTML.